AudioManagerUtilities/src/CAmSocketHandler.cpp

```diff
@@ -1020,30 +1083,43 @@ inline timespec* CAmSocketHandler::insertTime(timespec& buffertime)
         buffertime = mListActiveTimer.front().countdown;
         return (&buffertime);
     }
-    else
-#endif    
+    else   
+#endif         
     {
         return (NULL);
     }
 }
 
 #ifdef WITH_TIMERFD   
 am_Error_e CAmSocketHandler::createTimeFD(const itimerspec & timeouts, int & fd)
-{
+{    
     fd = timerfd_create(CLOCK_MONOTONIC, TFD_NONBLOCK | TFD_CLOEXEC);
-    if (fd <= 0)
+    if (fd < 0)
     {
         logError("Failed to create timer");
         return E_NOT_POSSIBLE;
     }
 
-    if (timerfd_settime(fd, 0, &timeouts, NULL))
+    if (timerfd_settime(fd, 0, &timeouts, NULL) < 0)
     {
         logError("Failed to set timer duration");
         return E_NOT_POSSIBLE;
     }
     return E_OK;
 }
+
+void CAmSocketHandler::closeRemovedTimers()
+{
+    std::list<sh_timer_s>::iterator it(mListRemovedTimers.begin());
+    while (it != mListRemovedTimers.end())
```

Comment:
Same here on **for** instead of **while**.  for is likely the most commonly used in the rest of the code base?